Index: pianobooster-src-0.6.7~svn156/src/QtMain.cpp
===================================================================
--- pianobooster-src-0.6.7~svn156.orig/src/QtMain.cpp
+++ pianobooster-src-0.6.7~svn156/src/QtMain.cpp
@@ -19,8 +19,10 @@
 
 ****************************************************************************/
 
-#include <QApplication>
+#include <GL/gl.h>
+#include <GL/glu.h>
 
+#include <QApplication>
 #include <QtOpenGL>
 #include "QtWindow.h"
 
Index: pianobooster-src-0.6.7~svn156/src/StavePosition.h
===================================================================
--- pianobooster-src-0.6.7~svn156.orig/src/StavePosition.h
+++ pianobooster-src-0.6.7~svn156/src/StavePosition.h
@@ -147,7 +147,7 @@ public:
 
 private:
     // fixme TODO This could be improved as the calculations could a done in the constructor
-    char m_staveIndex;    // 0 central line, 5 = top line, -5 the bottom line,
+    int m_staveIndex;    // 0 central line, 5 = top line, -5 the bottom line,
     int m_accidental;         // 0 = none, 1=sharp, -1 =flat, 2=natural
     float m_offsetY;
     whichPart_t m_hand;
