Description: this is not windows
Origin: vendor
Bug-Debian: http://bugs.debian.org/667322
Forwarded: no
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2012-05-23

Index: pianobooster-src-0.6.7~svn156/src/CMakeLists.txt
===================================================================
--- pianobooster-src-0.6.7~svn156.orig/src/CMakeLists.txt
+++ pianobooster-src-0.6.7~svn156/src/CMakeLists.txt
@@ -203,9 +203,9 @@ IF(WIN32)
     ADD_EXECUTABLE( pianobooster WIN32 pianobooster.rc ${PIANOBOOSTER_SRCS}
         ${PIANOBOOSTER_MOC_SRCS} ${PIANOBOOSTER_RC_SRCS}
         ${PIANOBOOSTER_UI_HDRS} )
+    SET_TARGET_PROPERTIES(pianobooster PROPERTIES LINK_FLAGS "-mwindows")
 ENDIF(WIN32)
 
-SET_TARGET_PROPERTIES(pianobooster PROPERTIES LINK_FLAGS "-mwindows")
 
 IF (USE_PCH)
 ADD_PRECOMPILED_HEADER( pianobooster ${CMAKE_CURRENT_SOURCE_DIR}/precompile/precompile.h )
