Index: pianobooster-src-0.6.7~svn156/music/CMakeLists.txt
===================================================================
--- pianobooster-src-0.6.7~svn156.orig/music/CMakeLists.txt
+++ pianobooster-src-0.6.7~svn156/music/CMakeLists.txt
@@ -6,7 +6,7 @@ SET( MUSIC_FILES
 ADD_CUSTOM_TARGET(music ALL DEPENDS ${MUSIC_FILES})
 
 IF (UNIX AND NOT APPLE)
-    INSTALL( FILES ${MUSIC_FILES} DESTINATION share/pianobooster/music )
+    INSTALL( FILES ${MUSIC_FILES} DESTINATION share/games/pianobooster/music )
 ENDIF (UNIX AND NOT APPLE)
 
 IF (WIN32)
Index: pianobooster-src-0.6.7~svn156/src/CMakeLists.txt
===================================================================
--- pianobooster-src-0.6.7~svn156.orig/src/CMakeLists.txt
+++ pianobooster-src-0.6.7~svn156/src/CMakeLists.txt
@@ -216,7 +216,7 @@ INSTALL( FILES pianobooster.desktop DEST
 INSTALL(TARGETS pianobooster RUNTIME DESTINATION bin)
 #INSTALL( index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/en  SUBDIR kmidimon )
 
-INSTALL( FILES ../README.txt  DESTINATION share/pianobooster/doc/ )
+INSTALL( FILES ../README.txt  DESTINATION share/games/pianobooster/doc/ )
 
 INSTALL ( FILES images/pianobooster.png DESTINATION share/pixmaps )
 
Index: pianobooster-src-0.6.7~svn156/src/QtMain.cpp
===================================================================
--- pianobooster-src-0.6.7~svn156.orig/src/QtMain.cpp
+++ pianobooster-src-0.6.7~svn156/src/QtMain.cpp
@@ -37,7 +37,7 @@ int main(int argc, char *argv[])
         QApplication::applicationDirPath() + "/translations/";
 #endif
 #ifdef Q_OS_LINUX
-        QApplication::applicationDirPath() + "/../share/" + QSTR_APPNAME + "/translations/";
+        QApplication::applicationDirPath() + "/../share/games/" + QSTR_APPNAME + "/translations/";
 #endif
 #ifdef Q_OS_DARWIN
         QApplication::applicationDirPath() + "/../Resources/translations/";
Index: pianobooster-src-0.6.7~svn156/src/Settings.cpp
===================================================================
--- pianobooster-src-0.6.7~svn156.orig/src/Settings.cpp
+++ pianobooster-src-0.6.7~svn156/src/Settings.cpp
@@ -422,7 +422,7 @@ void CSettings::unzipBootserMusicBooks()
         if (!QFile::exists(resourceDir + ZIPFILENAME))
         {
 #ifdef Q_OS_LINUX
-            resourceDir = QApplication::applicationDirPath() + "/../share/" + QSTR_APPNAME + "/music/";
+            resourceDir = QApplication::applicationDirPath() + "/../share/games/" + QSTR_APPNAME + "/music/";
 #endif
 #ifdef Q_OS_DARWIN
             resourceDir = QApplication::applicationDirPath() + "/../Resources/music/";
Index: pianobooster-src-0.6.7~svn156/translations/CMakeLists.txt
===================================================================
--- pianobooster-src-0.6.7~svn156.orig/translations/CMakeLists.txt
+++ pianobooster-src-0.6.7~svn156/translations/CMakeLists.txt
@@ -8,7 +8,7 @@ ADD_CUSTOM_TARGET(translations ALL DEPEN
 
 IF (UNIX AND NOT APPLE)
     INSTALL( FILES ${QM_FILES}
-             DESTINATION share/pianobooster/translations )
+             DESTINATION share/games/pianobooster/translations )
 ENDIF (UNIX AND NOT APPLE)
 
 IF (WIN32)
